src/tools/ScenarioMeasurement/Startup/Startup.cs

```diff
@@ -252,5 +252,18 @@ private static ProcessHelper CreateProcHelper(string command, string args, strin
             };
             return procHelper;
         }
+
+        private static void WriteResultTable(IEnumerable<Counter> counters, Logger logger)
+        {
+            logger.Log($"{"Metric",-15}|{"Average",-15}|{"Max",-15}|{"Min",-15}");
```

Comment:
Nit: I would propose that Min be written before max - this is commonly what folks expect.

Reply:
good point!